src/views/Home.vue

```diff
@@ -105,13 +105,13 @@ export default {
 .hero {
   position: relative;
   overflow: hidden;
-
   padding: 64px 0;
+  z-index: 0;
```

Comment:
I think default z-index is always 0 so this is not needed, but doesn't matter to much.

Reply:
i _think_ i started there but had to do this. Im probably doing something wrong. 😓